datadog_checks_dev/datadog_checks/dev/tooling/files/manifest.py

```diff
@@ -0,0 +1,78 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+import uuid
+
+from .utils import File
+
+TEMPLATE_IN = """\
+graft datadog_checks
+graft tests
+
+include MANIFEST.in
+include README.md
+include requirements.in
+include requirements.txt
+include requirements-dev.txt
+include manifest.json
+
+global-exclude *.py[cod] __pycache__
+"""
+
+TEMPLATE_JSON = """\
+{{
+  "display_name": "{check_name_cap}",
```

Comment:
This template is not accurate. See [doc](https://github.com/DataDog/integrations-core/blob/master/docs/dev/new_check_howto.md#manifestjson). Some of these fields do not exist anymore